packages/core/admin/server/bootstrap.js

```diff
@@ -92,6 +93,8 @@ module.exports = async () => {
   await syncAuthSettings();
   await syncAPITokensPermissions();
 
+  await revertSeatEnforcementWorkflow();
```

Comment:
I get why we're doing that, and sure it's needed, but still, it feels weird to use ee logic in the ce version :confused:
Can you share what were the other options available out of curiosity?

Reply:
Ideally during destroy() lifecycle, but that does not work properly